exercises/finale-workflow/README.md

```diff
@@ -0,0 +1,57 @@
+# Exercise 4: Finale Workflow
+One of the benefits of Temporal is that it provides SDKs for several
+languages and you can use multiple languages in the context of a single
+Workflow. For example, you might write your Workflow in Python but use
+Java or TypeScript for an Activity in that workflow. 
+
+The last exercise in this workshop does exactly that. The Workflow
+itself is written in Python, but the Activity that is executed as part
+of this Workflow is written in Java, as is the Worker that runs it.
+Since the Activity is written in Java, it's able to use a Java graphics
+library that would otherwise be would be incompatible with a typical 
+Python program. Since the Workflow and Activities are implemented in two 
+different languages, you'll need to start two Worker programs, one 
+written in Java that will execute the Activity code and one written 
+in Python that will execute the Workflow code.
+
+
+
+# Run the Activity Worker (implemented in Java):
```

Comment:
I think you'd get more mileage implementing in Go. It's often easier for people and is a good cross-language target.

Reply:
Unfortunately the PDF library necessary isn't available in Go. This was a decision made by Tom when writing the first 101 course.

Reply:
A bit rough to ask users to execute opaque compiled code and to ship around a JAR with samples, but meh